### DISCUSSION TITLE
discord server?

### DISCUSSION THREAD

**Discussion by nazthelizard122** [ggml-org/llama.cpp#250] [Ideas]
it would be awesome to start a discord server for easier discussion of llama (at least in my opinion) with separate channels for different topics

**Comment by prusnak** [comment]
I see very little added value. The discussions are already happening here on GitHub, where everybody already has an account and it is very easy to search in them.

**Comment by anzz1** [comment]
I too favor GitHub discussions and forums for better searchability and permanence. In general, while Discord is great for general chatting and memes, unfortunately the great migration from forums to Discord has already had the consequence of a ton of valuable information being 'lost'.

That being said, a Discord server could be great for general chat. But that again carries the penalty of losing the information. I can't even count the times I've found some valuable information in some random forum discussion which was posted a decade ago, which simply does not happen with Discord anymore.

**Comment by niansa** [comment]
I totally agree

**Comment by setzer22** [comment]
Since there doesn't seem to be interest in the llama.cpp community for a discord server, for people who have preference over Discord for more casual discussion feel free to join the [llama-rs discord server](https://discord.gg/NxgbV9mv). Note that the server is mainly focused in discussing the Rust fork of this repo, but general discussion and questions about the LLaMA models and any recent developments in the overall LLaMA ecosystem is both welcome and encouraged! :smile:  Keep in mind the server is not for asking technical support about the C++ code, but other than that, anyone is welcome to join.

**Reply by ggerganov** [reply]
If you like, you can add a Discord link to the README via PR - maybe a little badge at the top.
I personally will not hang too much if any on Discord, but it might be useful for other people to hang out.

**Reply by sloth-101** [reply]
invite link is no longer working

**Reply by limitedAtonement** [reply]
Please give new discord link

**Comment by ghost** [comment]
fwiw, there's also a (self-styled "Unofficial") llama.cpp Discord server on https://discord.gg/wnRNyrMJ6j (no specific focus, just a catch-all)

**Comment by sloth-101** [comment]
Oh thank your very much

Graham Higgins ***@***.***> schrieb am Di., 16. Mai 2023,
20:16:

**Comment by wtarreau** [comment]
Please no, don't move to discord. This is a completely closed discussion system where outsiders are not allowed to read contents nor participate, and it's not indexed in search engines. There's absolutely no value in burying useful information there, discussions must remain open just like the code. This project is popular precisely because it's totally open, and it should remain like this.

**Comment by clort81** [comment]
lol  banned from llama.cpp discord
wonder why
